src/main/java/seedu/lifeasier/commands/HelpCommand.java

```diff
@@ -22,6 +22,6 @@
     @Override
     public void execute(Ui ui, NoteList notes, TaskList tasks, FileStorage storage, Parser parser,
                         NoteHistory noteHistory, TaskHistory taskHistory) {
-        ui.showHelp();
+        ui.showHelpMessage();
```

Comment:
Maybe this would sound nicer, but i'm not sure if it will fit the current style you are going for.

```suggestion
        ui.showHelpMenu();
```

Reply:
I guess it's fine! Help is kind of an exception to the normal messages anyway.